src/darwin/Framework/CHIP/MTRDevice_Concrete.h

```diff
@@ -0,0 +1,33 @@
+/**
+ *
+ *    Copyright (c) 2022-2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <Matter/MTRDevice.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class MTRDeviceController;
+
+MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
+@interface MTRDevice_Concrete : MTRDevice
```

Comment:
"Concrete" doesn't really mean anything. Presumably other implementations of MTRDevice will also be "concrete"...

What's special about this impl?  Really, the fact that it does things with the Matter SDK directly, right?  So maybe MTRDeviceInProcess?  MTRDeviceDirect?  Having a hard time with the naming....

Reply:
`Direct` crossed my mind.

Reply:
Given this is a hidden class, we can rename it at any time, let's get this merged post tests passing, and please remove this availability flags here, since it's not public API